installation/local/bootstrap.sh

```diff
@@ -107,17 +106,20 @@ git clone --recurse-submodules --jobs 4 https://github.com/${GIT_DEFAULT_NAME}/C
 # International Neuroinformatics Coordinating Facility (INCF)
 # https://github.com/INCF/MUSIC
 # https://github.com/INCF/libneurosim
-git clone https://github.com/nest/nest-simulator.git ${NEST_SRC_CODE}
+git clone https://github.com/nest/nest-simulator.git
+cd nest-simulator
+git checkout 9cb3cb
```

Comment:
```suggestion
# 9cb3cb: Merge pull request from VRGroupRWTH/feature/device_label (https://github.com/nest/nest-simulator/commit/9cb3cb2ec1cc76e278ed7e9a8850609fdb443cae) 
# Needed until NEST v3.6 release to incorporate the aforementioned pull request.
git checkout 9cb3cb
```

Reply:
Thanks a lot! :+1: 
Could you maybe include something like `TODO` so that it can be found more easily?